pkg/domain/entities/containers.go

```diff
@@ -164,6 +164,7 @@ type ContainerStatReport struct {
 type CommitOptions struct {
 	Author         string
 	Changes        []string
+	Config         []byte
```

Comment:
also io.Reader here.

Reply:
Had to make it a `*io.Reader` to make the generator happy.